Client of WebTestCase throw error by test path like two slash '//' or more

### Symfony version(s) affected

4.4

### Description

Client of WebTestCase throw error by test path like two slash '//' or more:
`$client->request('GET', '//');`

Error messages
php 7.4: `array_replace(): Expected parameter 2 to be an array, bool given`
php 8.1: `TypeError: array_replace(): Argument #2 must be of type array, bool given`


Example console output on php 8.1
```
$ bin/phpunit 
PHPUnit 9.5.21 #StandWithUkraine

Testing 
E                                                                   1 / 1 (100%)

Time: 00:00.113, Memory: 20.00 MB

There was 1 error:

1) App\Tests\BugTest::testSomething
TypeError: array_replace(): Argument #2 must be of type array, bool given

/home/xxxxxxx/sf44_full/vendor/symfony/browser-kit/CookieJar.php:185
/home/xxxxxxx/sf44_full/vendor/symfony/browser-kit/Client.php:393
/home/xxxxxxx/sf44_full/tests/BugTest.php:12
```


### How to reproduce
```
<?php

namespace App\Tests;

use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;

class BugTest extends WebTestCase
{
    public function testSomething(): void
    {
        $client = static::createClient();
        $client->request('GET', '//');

        self::assertResponseStatusCodeSame(404);
    }
}
```


### Possible Solution

_No response_

### Additional Context

_No response_

--- a/src/Symfony/Component/BrowserKit/Client.php
+++ b/src/Symfony/Component/BrowserKit/Client.php
@@ -676,7 +676,7 @@ abstract class Client
         }
 
         // protocol relative URL
-        if (str_starts_with($uri, '//')) {
+        if ('' !== trim($uri, '/') && str_starts_with($uri, '//')) {
             return parse_url($currentUri, \PHP_URL_SCHEME).':'.$uri;
         }
 

--- a/src/Symfony/Component/BrowserKit/Tests/HttpBrowserTest.php
+++ b/src/Symfony/Component/BrowserKit/Tests/HttpBrowserTest.php
@@ -172,6 +172,30 @@ class HttpBrowserTest extends AbstractBrowserTest
         ]);
     }
 
+    /**
+     * @dataProvider forwardSlashesRequestPathProvider
+     */
+    public function testMultipleForwardSlashesRequestPath(string $requestPath)
+    {
+        $client = $this->createMock(HttpClientInterface::class);
+        $client
+            ->expects($this->once())
+            ->method('request')
+            ->with('GET', 'http://localhost'.$requestPath)
+            ->willReturn($this->createMock(ResponseInterface::class));
+        $browser = new HttpBrowser($client);
+        $browser->request('GET', $requestPath);
+    }
+
+    public function forwardSlashesRequestPathProvider()
+    {
+        return [
+            'one slash' => ['/'],
+            'two slashes' => ['//'],
+            'multiple slashes' => ['////'],
+        ];
+    }
+
     private function uploadFile(string $data): string
     {
         $path = tempnam(sys_get_temp_dir(), 'http');

